colossalai/zero/utils/tensor_placement_policy.py

```diff
@@ -70,7 +70,7 @@ def evict_tensors(self,
             freed_cuda_model_data = 0
             to_free_tensor_list = hold_cuda_tensor_list
             if not warmup:
-                next_compute_idx: Dict[StatefulTensor, int] = {t: len(compute_list) for t in hold_cuda_tensor_list}
```

Comment:
Why did you delete the type hint?

Reply:
Because `next_compute_idx` become List[StatefulTensor, int] below.
This type hint is not consistent.